submodules/LIB/scripts/caravel_gds.py

```diff
@@ -0,0 +1,8 @@
+import os
```

Comment:
This seems to be a build script -- use only one build script and one setup script